cv32e40s/bsp/Makefile

```diff
@@ -4,11 +4,11 @@ RISCV_EXE_PREFIX ?= $(RISCV)/bin/riscv32-unknown-elf-
 RISCV_CC         ?= gcc
 RISCV_GCC = $(RISCV_EXE_PREFIX)$(RISCV_CC)
 RISCV_AR = $(RISCV_EXE_PREFIX)ar
-RISCV_MARCH ?= rv32imc
+RISCV_MARCH ?= rv32im_zca
 SRC = crt0.S handlers.S syscalls.c vectors.S
 OBJ = crt0.o handlers.o syscalls.o vectors.o
 LIBCV-VERIF = libcv-verif.a
-CFLAGS ?= -Os -g -static -mabi=ilp32 -march=$(RISCV_MARCH) -Wall -pedantic $(RISCV_CFLAGS)
+CFLAGS ?= -Os -g -static -mabi=ilp32 -march=$(RISCV_MARCH) -Wall -pedantic -mno-relax $(RISCV_CFLAGS)
```

Comment:
Could you explain why the no-relax does and why it is needed?

Reply:
This has resolved conflicts I've seen with Zcmt enabled (tablejumps depend on linker relaxation) - It might not be necessary when the toolchain issues are sorted out, but it does not hurt, and makes the compiled code somewhat more straightforward to debug.